_pages/en_US/homebrew-launcher.txt

```diff
@@ -23,6 +26,16 @@ We launch this using the Wii U's built in browser, so your Wii U will need to be
   + If it freezes, just force the console to power off by holding the power button, then try again
 1. Your console should load the Homebrew Launcher
 
+### For 5.5.2
+
+1. Go to `https://u.drg.li/`
```

Comment:
This site does not have a valid SSL certificate (https gives an error, use http).